includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -493,7 +493,7 @@ public function payment_fields() {
 				<div id="wc-stripe-upe-errors" role="alert"></div>
 				<input id="wc-stripe-payment-method-upe" type="hidden" name="wc-stripe-payment-method-upe" />
 				<input id="wc_stripe_selected_upe_payment_type" type="hidden" name="wc_stripe_selected_upe_payment_type" />
-				<input type="hidden" id="wc-stripe-is-deferred-intent" name="wc-stripe-is-deferred-intent" value="1" />
+				<input type="hidden" class="wc-stripe-is-deferred-intent" name="wc-stripe-is-deferred-intent" value="1" />
```

Comment:
If cards were disabled, the core `WC_Stripe_UPE_Payment_Gateway::payment_fields()` wouldn't run and so wouldn't include this hidden element. I've added it to all our UPE method payment fields so it should now always be included. 

Its my understanding that these may be removed anyway once we no longer need the deferred intent flag.